### DISCUSSION TITLE
Help with Google Chronicle Configuration for Non-Default Log Sources

### DISCUSSION THREAD

**Discussion by TommySarkissian** [vectordotdev/vector#17284] [Q&A] [answered]
Would anyone happen to have a Google Chronicle configuration that I could use as a reference? I'm uncertain about how to utilize the sink for sources that don't employ a default parser. For instance, I obtain logs from a SaaS product using an HTTP GET request. In situations where there is no default supported parser or log type, how would I push those logs?

**Comment by StephenWakely** [comment, marked as answer]
Unless I am misunderstanding your question, I think this is more a question that needs to be directed at Chronicle. I believe you will need to setup a parser there that can parse your logs. It looks like [this](https://cloud.google.com/chronicle/docs/preview/parser-extensions/parsing-overview) could be a useful starting point.

There are things you can do in Vector using the [Remap](https://vector.dev/docs/reference/configuration/transforms/remap/) transform to modify your data if you need to structure it in a way that can make it easier to parse in Chronicle. That will very much depend on what the current structure is, if you have any questions after reading the docs, please ask.
